stormworkflow/post/analyze_ensemble.py

```diff
@@ -408,19 +408,19 @@ def cli():
     parser.add_argument('-t', '--tracks-dir', type=Path)
     parser.add_argument('-s', '--sequential', action='store_true')
 
-    main(parser.parse_args())
-
-
-if __name__ == '__main__':
     cluster = SLURMCluster(cores=16,
                            processes=1,
                            memory="500GB",
-                           account="compute",
+                           account="nos-surge", #PW:"compute" ; Hercules:"nos-surge" or "nosofs"
                            walltime="08:00:00",
-                           header_skip=['--mem'],
-                           interface="eth0") 
-    cluster.scale(6) 
-    client = Client(cluster) 
+                           header_skip=['--mem']) #,
+#                           interface="eth0")     # only for PW
```

Comment:
For future reference, it's better to remove these platform specific hardcoded values and get the values from the `input.yaml` file, similar to what we do for HPC account and solver ntasks:

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/refs/input.yaml#L18-L21

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/scripts/workflow.sh#L42-L47

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/scripts/workflow.sh#L168-L173

Reply:
For now I'll merge this to fix the issue with the location of cluster creation within the script

Reply:
@FariborzDaneshvar-NOAA when you have time, can you please create another ticket about removal of those hardcoded values?

Reply:
Sure, I will.